### PR TITLE
Multiple improvements

### PR DESCRIPTION
**Pull request by kaizo**
- Tw translations
- Light theme
- Fix reconnection
- Logout notification

### CODE REVIEW

File: alpine-dependencies-dev.list [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Install packages to images docker, "common" is declared into dependencies-extra of alpine6-node-base.
+add="$common autoconf automake bash bzip2 libffi-dev libpng libpng-dev nasm nodejs=4.3.0-r0 ruby ruby-dev"
+
+# Delete packages to images docker, "common" is declared into dependencies-extra of alpine6-node-base.
+del="$common autoconf automake bzip2 libffi-dev libpng libpng-dev ruby ruby-dev"
```
Comment (jordins) [inline alpine-dependencies-dev.list]: This file is almost duplicated. As we've talked before we can execute always the dependencies-extra and later the dependencies-dev to remove duplication.

File: app/addons/applications-workspace/sass/_variables.scss [hunk 2]
```diff
@@ -58,6 +58,7 @@ $icon-shadow: ' ';
 	$icon-chrome-color: #fbcb00;
 	$icon-firefox-color: #e17300;
 	$icon-explorer-color: #2ebaec;
+	$icon-gymp-color: #AA6C39;
```
Comment (jordins) [inline app/addons/applications-workspace/sass/_variables.scss]: typo: gimp

File: app/addons/applications-workspace/sass/_variables.scss [hunk 3]
```diff
@@ -78,6 +79,7 @@ $icon-shadow: ' ';
 	$icon-chrome-char: '\00c8';
 	$icon-firefox-char: '\00d2';
 	$icon-explorer-char: '\00c7';
+	$icon-gymp-char: '\002a';
```
Comment (jordins) [inline app/addons/applications-workspace/sass/_variables.scss]: typo: gimp

File: app/modules/cloud/cloud.js [hunk 4]
```diff
@@ -41,6 +41,23 @@ define([
                 cloudFileOpenService.start();
                 return cloudFileOpenService;
         }])
+        .controller('topbarController', ['$scope', function ($scope) {
```
Comment (jordins) [inline app/modules/cloud/cloud.js]: could you move it to a separate file?

File: app/modules/cloud/cloud.js [hunk 5]
```diff
@@ -78,7 +96,8 @@ define([
                         mail: "Mail",
                         calc: "Spreadsheet",
                         presentation: "Presentation",
-                        writer: "Writer"
+                        writer: "Writer",
+                        gymp: "Gymp"
```
Comment (jordins) [inline app/modules/cloud/cloud.js]: typo

File: app/modules/logoutNotifier/logoutNotifierController.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+		this.$translate = $translate;
+		this.bootstrapDialog = bootstrapDialog || window.BootstrapDialog;
+	}
+
+	LogoutNotifierController.prototype.init = function () {
+		this.logoutNotifierService.addLogoutCallback(this.notifyLogout.bind(this));
+	};
+
+	LogoutNotifierController.prototype.notifyLogout = function () {
+		$('body').addClass('modal_mode_on');
```
Comment (jordins) [inline app/modules/logoutNotifier/logoutNotifierController.js]: Could we have this "addClass" and "show" function grouped in a same method?
I mean, create a eyeosDialog object with a showModal method which:

1. Adds the class modal
2. calls bootstrapDialog.show

File: app/modules/logoutNotifier/logoutNotifierService.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+		function handleVisibilityChange() {
+			if (!document[hidden]) {
+				if (!window.localStorage.card || !window.localStorage.signature) {
+					self.notifyLogout();
+				}
+			}
+		}
+
+		document.addEventListener(visibilityChange, handleVisibilityChange, false);
```
Comment (jordins) [inline app/modules/logoutNotifier/logoutNotifierService.js]: All this visibility api part should be in its own class. This service should only call this new class with its own handler.

File: app/themes/eyeos-cloud-light/info.json [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    "popupStartMenu": "popupTopbar/startMenu.tpl.html",
+    "main": "desktop/desktop.tpl.html",
+    "dockbar": "dockbar/dockbar.tpl.html",
+    "topbar": "topbar/topbar.tpl.html",
+    "welcome": "welcome/welcome.tpl.html",
+    "chat": "chat/chat.tpl.html",
+    "videoconference": "videoconference/videoconference.tpl.html",
+    "startMenu": "topbar/startMenu.tpl.html"
+  },
+  "extends": "workspacev2",
```
Comment (jordins) [inline app/themes/eyeos-cloud-light/info.json]: @kaizo why can't we extend from "cloud"?

File: locales/loc.desktop.en_EN.po [hunk 9]
```diff
@@ -410,6 +410,10 @@ msgid "Presentation"
 msgstr "Impress"
 
 #:
+msgid "Gymp"
```
Comment (jordins) [inline locales/loc.desktop.en_EN.po]: typo

File: package.json [hunk 10]
```diff
@@ -31,7 +31,7 @@
     "grunt-karma": "2.0.0",
     "grunt-newer": "0.7.0",
     "grunt-ngmin": "0.0.3",
-    "grunt-svgmin": "0.4.0",
+    "grunt-svgmin": "^3.2.0",
```
Comment (jordins) [inline package.json]: Please, fix the version

Comment (absis) [inline package.json]: DONE

File: app/modules/logoutNotifier/logoutNotifierService.js [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+		} else if (typeof document.webkitHidden !== "undefined") {
+			hidden = "webkitHidden";
+			visibilityChange = "webkitvisibilitychange";
+		}
+
+		var self = this;
+
+		function handleVisibilityChange() {
+			if (!document[hidden]) {
+				if (!window.localStorage.card || !window.localStorage.signature) {
```
Comment (jordins) [inline app/modules/logoutNotifier/logoutNotifierService.js]: Could you use eyeosAuthClient.getCredentials ?

File: bower.json [hunk 12]
```diff
@@ -8,6 +8,7 @@
     "json3": "3.3.1",
     "es5-shim": "3.1.0",
     "jquery": "2.1.1",
+    "jquery-visibility": "1.0.11",
```
Comment (jordins) [inline bower.json]: @kaizo I can't see where we are loading this file (Maybe you forgot to require it)

**Pull request merged**